tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -0,0 +1,263 @@
+using System;
```

Comment:
Consider renaming this file to `Build.TestMatricesGeneration.cs`

Reply:
Class is used for generating matrices and condition variables, and can be used for any step, so  so TestMatrixGeneration is not enough.

Reply:
Fair point, this rename would only make sense based on what you think about my other comment, that we should split those two things apart 🙂

Reply:
We can split targets apart, however we have a lot of targets, so this only ads complexity.
So, I prefer to leave it's name for now, since I don't have a better name for this.

Reply:
Renamed to `Build.VariableGenerations`
I've extracted condition generation to separate method, but same target for now.
It can be extracted to target at any point of time.